app/components/AdditionalData/AdditionalDataSettings.tsx

```diff
@@ -47,16 +47,18 @@ const AdditionalDataSettings = () => {
     };
 
     Share.open(options).catch((err) => {
-      setAlertHeading(i18next.t('label.something_went_wrong'));
-      setAlertMessage(i18next.t('label.share_additional_data_error'));
-      setShowSecondaryButton(false);
-      setShowAlert(true);
+      if (err?.error?.code != 'ECANCELLED500') { // iOS cancel button pressed
```

Comment:
I am not sure if we can rely on this code for future version of iOS, but this is what my iOS simulator returns when I cancel the sharing dialog.

Reply:
I could also successfully test this on iOS 14.6 with my iPhone X.